### PR TITLE
New events should default to Lecture

### PR DESCRIPTION
**Pull request by bp323**
As it says on the tin, all new events created either via
* [x] batch edit
* [x] new event button
* [x] add new events in empty state
* [x] overflowing OT events

Should have a defined event type as Lecture.

### CODE REVIEW

**Comment by bp323** [comment]
Let's pull this one into Alpha as well

**Comment by bp323** [comment]
Assigning to @Coenego for review.

File: shared/gh/js/views/gh.admin-batch-edit-date.js [hunk 1]
```diff
@@ -169,7 +169,7 @@ define(['lodash', 'moment', 'gh.core', 'gh.api.config'], function(_, moment, gh,
                         'displayName': $('.gh-jeditable-series-title').text(),
                         'end': gh.utils.convertUnixDatetoISODate(moment(endDate).toISOString()),
                         'location': '',
-                        'notes': 'Lecture',
+                        'type': 'Lecture',
```
Comment (Coenego) [inline shared/gh/js/views/gh.admin-batch-edit-date.js]: Maybe we can use the default value from the configuration here: `gh.config.events.default`

File: shared/gh/js/views/gh.admin-batch-edit-date.js [hunk 2]
```diff
@@ -231,7 +231,7 @@ define(['lodash', 'moment', 'gh.core', 'gh.api.config'], function(_, moment, gh,
                                     'displayName': $('.gh-jeditable-series-title').text(),
                                     'end': gh.utils.convertUnixDatetoISODate(moment(endDate).toISOString()),
                                     'location': '',
-                                    'notes': 'Lecture',
+                                    'type': 'Lecture',
```
Comment (Coenego) [inline shared/gh/js/views/gh.admin-batch-edit-date.js]: Same here, better make us of `gh.config.events.default`

**Comment by Coenego** [comment]
Reassigning to @bp323 for follow-up

**Comment by bp323** [comment]
Back to @Coenego

**Pull request merged**